tests/sysauth/sssd.pm

```diff
@@ -29,10 +29,12 @@ sub run {
     if (is_sle) {
         assert_script_run 'source /etc/os-release';
         if (is_sle '>=15') {
-            if (script_run('SUSEConnect -p PackageHub/${VERSION_ID}/${CPU}', 300) != 0) {
-                record_soft_failure 'bsc#1124318 - Fail to get PackageHub Pool Metadata - running the command again as a workaround';
-                assert_script_run 'SUSEConnect -p PackageHub/${VERSION_ID}/${CPU}', 300;
-            }
+            my $version = '${VERSION_ID}';
```

Comment:
Sorry for requesting changes again, it seems my review was sent incomplete.
I also was asking for a comment to explain why we use `'${VERSION_ID}'`.
For example:

```suggestion
            # Reusing bash variables that are set on https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/5aa34ae4ae7bc651cf0a01c44d513935548c87ce/lib/registration.pm#L149
            my $version = '${VERSION_ID}';
```